db/migrate/20150421161752_add_disabled_to_user.rb

```diff
@@ -0,0 +1,5 @@
+class AddDisabledToUser < ActiveRecord::Migration
+  def change
+    add_column :users, :disabled, :boolean
```

Comment:
Good spot. I've also added the shared_example from gds-sso to provide some validation of the model, although it didn't catch this particular problem.